pyaedt/application/Design.py

```diff
@@ -2168,29 +2224,37 @@ def export_variables_to_csv(self, filename, export_project=True, export_design=T
 
     @aedt_exception_handler
     def read_design_data(self):
-        """Reads back the design data as a dictionary"""
+        """Read back the design data as a dictionary.
+        
+        Returns
+        -------
+        dict
+        Dictionary of design data.
+        
+        """
         design_file = os.path.join(self.working_directory, "design_data.json")
         with open(design_file, 'r') as fps:
             design_data = json.load(fps)
         return design_data
 
     @aedt_exception_handler
     def save_project(self, project_file=None, overwrite=True, refresh_obj_ids_after_save=False):
-        """Save the AEDT Project and add a message.
+        """Save the AEDT project and add a message.
 
         Parameters
         ----------
         project_file : str, optional
-            Full Project path. Default ``None``.
+            Full path and project file name. The default is ````None``.
         overwrite : bool, optional
-            Overwrite existing project.  Default ``True``.
-        refresh_obj_ids_after_save : bool
-            Refresh object IDs after save.  Default ``False``.
+            Whether to overwrite the existing project. The default is ``True``.
+        refresh_obj_ids_after_save : bool, optional
+            Whether to refresh object IDs after saving the project.  
+            The default is ``False``.
 
         Returns
         -------
         bool
-            Always returns ``True``
+            Always returns ``True``.
```

Comment:
I don't remember seeing anything like this. Is it correct?

Reply:
Returning nothing means that we don't need the returns section.

As for "always returns `True`", this appears to be incorrect.  The method has the aedt exception decorator, meaning that it's still going to return true/false depending on if it succeeds.